scripts/common/addon.sh

```diff
@@ -62,7 +62,7 @@ function addon_fetch() {
     if [ "$AIRGAP" != "1" ]; then
         if [ -n "$s3Override" ]; then
             rm -rf $DIR/addons/$name/$version # Cleanup broken/incompatible addons from failed runs
-            addon_fetch_no_cache "$s3Override"
+            addon_fetch_cache "$name-$version.tar.gz" "$s3Override"
```

Comment:
is it ok to fetch s3Override with cache?

Reply:
I think it's fine
we repull if the sha changed, right?

Reply:
ya. looks like we check the etag in s3